vaadin-login-flow-integration-tests/src/test/java/com/vaadin/flow/component/login/test/BasicIT.java

```diff
@@ -43,6 +43,18 @@ public void forgotPassword() {
                 notification);
     }
 
+    @Test
+    public void disabledLogin() {
```

Comment:
Do you think it makes sense to test the submit action when the user hits the ENTER key?

Reply:
Would be nice, but not sure `sendKeys` would work in firefox